### ISSUE TITLE
Add popular program ids

### ISSUE THREAD

**Issue by acheroncrypto** [solana-playground/solana-playground#101]
It would be nice to have all popular program ids in a constant that's globally available in playground client.

For example: `PROGRAM_ID.SPL_MEMO` to access [spl-memo](https://github.com/solana-labs/solana-program-library/tree/master/memo)'s program id.

**Comment by dhruvja** [comment]
You mean constants while writing client code or tests?

**Comment by acheroncrypto** [comment]
Yes, that was the original intent back then but now that I look at it again, I think we can support popular programs' npm packages instead. That being said, we can still use popular program ids in the test UI.

**Comment by arihantbansal** [comment]
I'd like to work on this. I shall make a list of the ones I find relevant and post them here to confirm.
